pkgs/kubepug/kubepug/registry.yaml

```diff
@@ -3,19 +3,66 @@ packages:
   - type: github_release
     repo_owner: kubepug
     repo_name: kubepug
-    aliases:
-      - name: rikatz/kubepug
```

Comment:
aliases was removed.

Reply:
Oops! I noticed that but forgot to add back, fixing in a jiffy

Reply:
fecddef7202a81fca00720534bc18dc5357b0261